_sass/variables/_colors.scss

```diff
@@ -1,82 +1,111 @@
+// Top-level color variables.
+// [color|backgroundColor|etc] - [...modifiers]
+$backgroundColor: white !default;
+
+$color-dark-highContrast: black !default;
+$color-light-highContrast: white !default;
+$color-highlight-highContrast: yellow !default;
+$color-success-highContrast: green !default;
+$color-danger-highContrast: red !default;
+$color-medium-highContrast: #767676 !default;
+
+$color-danger: #E27874 !default;
+$color-warning: #f0ad4e !default;
+$color-success: #5cb85c !default;
+
+// Element-level color variables.
+// [element] - [color|backgroundColor|etc] - [...modifiers]
 $goal-colors: #E52B3D #DDA63A #4C9F38 #C42130 #FF3E24 #28BCE1 #FCC30B #A21942 #FD6925 #DD1367 #FD9C27 #BF8B2E #417D46 #0A96D8 #57BE2F #00689C #1F4A6A !default;
-$goal-banner-background-colors: white white white white white white white white white white white white white white white white white !default;
 
 $text-color: #333 !default;
-$text-dark-color: black !default;
-$text-on-dark-background-color: white !default;
-$background-color: white !default;
+$text-color-dark: black !default;
+$text-color-light: white !default;
+
```

Comment:
Would these belong to the "top-level" variables? Also I think `color` implies that it's for text coloring.

Reply:
Should we make them `color-dark` and `color-light` instead?

Reply:
Trying to figure out under what circumstances this colours are used. It seems to be a widely used colour, but not as "top-level", but rather when there's a box that has text in it, like the "ALPHA" box at the very top or the indicator statuses. Is this correct?

Reply:
Yes `text-color-light` (or `color-light` if we change it) would be for any situation where the background is dark.

Reply:
That makes sense. I think it would be helpful if you specify that this is for those specific elements. Not sure what they're called. Phases? What do you think?

Reply:
Sounds good, I just made that change, and the color-dark/color-light change.

Reply:
Looks good. Are the `color-dark` and `color-light` variables more general or are they specific to the disclaimers/phases as well? I'm under the impression that they are.

Reply:
@SavvasStephanides Right, they are more general.